### PR TITLE
Detect iSCSI devices

### PR DESCRIPTION
**Pull request by avishayt**
Fixes: https://github.com/jaypipes/ghw/issues/297

### CODE REVIEW

File: pkg/block/block_linux.go [hunk 1]
```diff
@@ -277,6 +277,9 @@ func disks(ctx *context.Context, paths *linuxpath.Paths) []*Disk {
 		size := diskSizeBytes(paths, dname)
 		pbs := diskPhysicalBlockSizeBytes(paths, dname)
 		busPath := diskBusPath(paths, dname)
+		if strings.Contains(busPath, "-iscsi-") {
+			driveType = DRIVE_TYPE_ISCSI
```
Comment (fromanirh) [inline pkg/block/block_linux.go:281]: I wonder if there is another way to know if a device is iscsi or not

**Review by fromanirh — COMMENTED** [review]
thanks for the PR! indeed acknowledging ISCSI devices would be nice.
It would be real cool to add unit test(s) to exercise this code, even if it's a simple change, to make it unlikely to break in the future

File: pkg/block/block_linux.go (hunk 1 above)
Comment (avishayt) [reply pkg/block/block_linux.go:281]: Not that I know of.  This method has worked consistently for me.

**Comment by avishayt** [comment]
> thanks for the PR! indeed acknowledging ISCSI devices would be nice. It would be real cool to add unit test(s) to exercise this code, even if it's a simple change, to make it unlikely to break in the future

Added a unit test - thanks!

**Comment by fromanirh** [comment]
thanks for the updates! codewise LGTM - we can further improve the testing in the future if needed/feasible.
The only thing left is the DCO check which is failing - could you PTAL to the instructions (https://github.com/jaypipes/ghw/pull/300/checks?check_run_id=5556037529) and see if there is a easy fix? feel free to get in touch if instructions are not clear

**Comment by avishayt** [comment]
> thanks for the updates! codewise LGTM - we can further improve the testing in the future if needed/feasible. The only thing left is the DCO check which is failing - could you PTAL to the instructions (https://github.com/jaypipes/ghw/pull/300/checks?check_run_id=5556037529) and see if there is a easy fix? feel free to get in touch if instructions are not clear

Fixed, thanks for the pointer

File: pkg/block/block_linux.go (hunk 1 above)
Comment (fromanirh) [reply pkg/block/block_linux.go:281]: We can surely start like this

**Pull request closed without merging**